frontend/src/scenes/surveys/SurveySettings.tsx

```diff
@@ -7,14 +7,61 @@ import { useState } from 'react'
 import { surveysLogic } from 'scenes/surveys/surveysLogic'
 import { sanitizeSurveyAppearance, validateColor } from 'scenes/surveys/utils'
 import { teamLogic } from 'scenes/teamLogic'
+import { urls } from 'scenes/urls'
 
 import { SurveyAppearance } from '~/types'
 
 import { defaultSurveyAppearance, NEW_SURVEY } from './constants'
 import { SurveyAppearancePreview } from './SurveyAppearancePreview'
 import { Customization } from './SurveyCustomization'
 
-export function SurveySettings(): JSX.Element {
+interface Props {
+    isModal?: boolean
+}
+
+function SurveyPopupToggle(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const { updateCurrentTeam } = useActions(teamLogic)
+
+    return (
+        <div className="flex flex-col gap-1">
+            <LemonSwitch
+                data-attr="opt-in-surveys-switch"
+                onChange={(checked) => {
+                    updateCurrentTeam({
+                        surveys_opt_in: checked,
+                    })
+                }}
+                fullWidth
+                bordered={false}
+                label="Enable surveys popup"
```

Comment:
```suggestion
                label="Enable surveys"
```

since this is not only about the popup right? remote config and decide API actually checks if surveys is enabled (or will check after my PR)

Reply:
even if that is disabled, API surveys still work, unless i'm missing something? because you can still create surveys with the API and their data will show up as we store events. just not surveys that rely on popup, like widget and the actual popup survey

Reply:
> their data will show up as we store events
that is fine, if they used in the past, they should still be able to see their results

> API surveys still work
correct, if people/SDKs are calling the APIs directly, its fine.

if people are using remote config/decide API as our SDKs, surveys wont be returned after ->
https://github.com/PostHog/posthog/pull/29637/files
so no surveys will be returned even if created by the API, we will stop using the surveys API directly and only use the remote config API for loading surveys
This is how mobile works, and how JS should work as well after https://github.com/PostHog/posthog-js/pull/1652

Reply:
yeah no surveys will be returned on the SDK, but, for API surveys, the surveys part of the SDK is not needed. as you just need to capture events with that specific meta data, and not really check if surveys are being returned or not.

Reply:
well popup is != than the feedback button, that alone will bring confusion, is it enabled for popup and not for feedback button vice-versa.
all configurations there in the project settings page don't mention something specific, and you still could use their API directly as well (eg session recording), this is a remote config API for the SDKs.
your call here, I prefer the `popup` out.

Reply:
IMO even the APIs should not return anything if its disabled, that's the whole point of the remote config, but its a huge breaking change so I've not done that, just the remote config and decide API

Reply:
> To me it's an all or nothing config.

This is not what all the other settings do, so I don't think we should all/nothing.
The other settings do: eg 'Record user sessions', 'Capture console logs', etc.

Another example is 'Enable exception autocapture', you can still capture exceptions manually, this is more about the automatic behaviour.
Its ok that the API returns surveys if you call it directly. 

It was 'Enable surveys popup' already, so I guess the confusion isn't an issue otherwise, this would have come up already.

Reply:
i agree with all the thoughts here, also changed the label. Also think we should gate the API functionality behind things, but I understand the rationale if it's too big of a change